pages/windows/color.md

```diff
@@ -14,3 +14,17 @@
 - Set the console foreground and background to a specific color:
 
 `color {{foreground_code}}{{background_code}}`
+
+Allowed values: `1-9,a-f`
+| 1-9           | a-f             |
+|---------------|-----------------|
+| 0: Black      | a: L Green      |
+| 1: Blue       | b: L Aqua       |
+| 2: Green      | c: L Red        |
+| 3: Aqua       | d: L Purple     |
+| 4: Red        | e: L Yellow     |
+| 5: Purple     | f: Bright White |
+| 6: Yellow     |                 |
+| 7: White      |                 |
+| 8: Gray       |                 |
+| 9: Light Blue |                 |
```

Comment:
```suggestion
```

Reply:
This doesn't fit the syntax or style of TLDR unfortunately, therefore it needs to be removed.

Reply:
As @CleanMachine1 says, while this is a great table we have a strict linter for tldr pages, so this will unfortunately have to be removed. It would be a great fit for a cheat sheet project though! We have a number of similar projects listed in our README. Perhaps it would be a good fit for one or more of those?